CMakeLists.txt

```diff
@@ -274,17 +222,17 @@ set_property(TARGET monero::libraries PROPERTY
     ${Boost_SERIALIZATION_LIBRARY}
     ${Boost_SYSTEM_LIBRARY}
     ${Boost_THREAD_LIBRARY}
-    ${HIDAPI_LIBRARIES}
-    ${OPENSSL_LIBRARIES}
+    ${monero_HIDAPI_LIBRARY}
+    ${monero_usb_LIBRARY}
+    ${monero_LIBUDEV_LIBRARY}
+    ${ZMQ_LIB}
```

Comment:
Is this needed here? It might be needed in `monerod` 0.18+ now, but previously I got away with only linking against ZeroMQ in the binaries that used it (`monero-lws-admin` technically shouldn't need it, I think).

Reply:
Good spot, builds and runs fine without it. Guessing it was missed how it's already set for `monero-lws-daemon` in `src/CmakeLists.txt`. Removed